examples/guided.py

```diff
@@ -58,38 +70,44 @@ def ask_user_questions():
 		will we continue with the actual installation steps.
 	"""
 
-	global_menu = archinstall.GlobalMenu()
-	global_menu.enable('keyboard-layout')
-
+	# #
 	if not archinstall.arguments.get('ntp', False):
 		archinstall.arguments['ntp'] = input("Would you like to use automatic time synchronization (NTP) with the default time servers? [Y/n]: ").strip().lower() in ('y', 'yes', '')
 		if archinstall.arguments['ntp']:
 			archinstall.log("Hardware time and other post-configuration steps might be required in order for NTP to work. For more information, please check the Arch wiki.", fg="yellow")
 			archinstall.SysCommand('timedatectl set-ntp true')
 
-	# Set which region to download packages from during the installation
-	global_menu.enable('mirror-region')
+	global_menu = archinstall.GlobalMenu(pos_callback=mnu_post_callback,exit_callback=_post_processing)
```

Comment:
> I'm a bit confused here, why is the `_post_processing` here passed as the `exit_callback`? Shouldn't this be rather the actual `pos_callback`?

NO. if it where a *pos_callback*, it would be executed after EACH selection, but the code in this case (updating some archinstall.arguments fields)  must only be executed ONCE at the exit of the loop, and is application dependant. Probably the confusion springs from the name *_post_processing*; I just inherited it from your original, where it made totally sense. I'll look to change the name to something less confusing